packages/quantic/cypress/e2e/default-2/result-list/folded-result-list/folded-result-list.cypress.ts

```diff
@@ -32,150 +39,164 @@ describe('quantic-folded-result-list', () => {
   const defaultChildField = 'foldingparent';
   const defaultNumberOfFoldedResults = 2;
   const registerResultTemplatesEvent = 'registerresulttemplates';
-  const useCase = 'search';
 
   function visitFoldedResultList(
     options: Partial<FoldedResultListOptions> = {},
     withoutChildResults = false
   ) {
     interceptSearch();
     if (!withoutChildResults) {
-      mockSearchWithResults(exampleResult.data);
+      mockSearchWithResults(exampleResult.data, options.useCase);
     } else {
-      mockSearchWithResults();
+      mockSearchWithResults([{}], options.useCase);
```

Comment:
this I put [{}] instead of an array because the type of result is results?: Array<object>
I thought it would be maybe clearer for future dev working in the file but then again I can put [] if you judge its better?

Reply:
what would be best is this: `[{title: 'Result', uri: 'uri', raw: {uriHash: 'resulthash'}}]`
Cause that's exactly what we are trying to mock here, a result that doesn’t have child results.